PLAN.md

```diff
@@ -1,12 +1,10 @@
 # Xenon Cybersecurity Plan
 
-*Updated February 8th, 2019.*
+*Updated February 7, 2020.*
 
 ## Schedule
 
-This document is reviewed quarterly. The next review will be in **Q2 2019**. At this review:
- * Business Continuity Plans are to be reviewed and tested. (CAIQ: BCR-02)
- * Application and infrastructure [security baseline](PRACTICES.md#External_Monitoring) to be reviewed. (CAIQ: GRM-01)
+This document is reviewed quarterly.
```

Comment:
I like the date because it exposes any inability for us to achieve our commitment.